java/src/Ice/src/main/java/com/zeroc/Ice/InputStream.java

```diff
@@ -962,17 +964,17 @@ public byte readByte()
      * Extracts an optional byte value from the stream.
      *
      * @param tag The numeric tag associated with the value.
-     * @return The optional value (if any).
+     * @return The extracted byte value, or null if it wasn't sent.
```

Comment:
I don't like `sent` here: it's null when there is no entry for the provided tag in the tagged members map. 
Maybe the simplest is `The extracted Byte`, and it's obvious from the API it's nullable.

In the future, we'll probably have 3 overloads:
 - byte readByte() // non-nullable byte   (supported by all Ice encodings)
 - @Nullable Byte readNullableByte() // nullable Byte (supported only by Ice encoding >= 2.0)
 - @Nullable Byte readNullableByte(int tag) // nullable Byte (supported only by Ice encoding >= 1.1)